components/dialog/dialog-styles.js

```diff
@@ -11,6 +11,15 @@ export const dialogStyles = css`
 		display: block;
 	}
 
+	:host([opened]:not([_state="showing"])) {
+		visibility: hidden;
+	}
+
+	:host([opened][_state="showing"]),
+	:host([opened][_state="hiding"]) {
+		visibility: visible;
+	}
```

Comment:
This is necessary because of the timing change from the switch from rAF -> setTimeout. Before the dialog finishes opening (i.e. before the state transition starts), there's a brief moment where the dialog pops in at max width before the transition starts and the backdrop gets applied. This additional CSS ensures that the dialog only becomes visible once we start the state transition.

Reply:
Ah, sorry, wasn't clear. It sets `width: auto`, which in most cases fills the viewport.